src/main/java/uk/gov/companieshouse/api/accounts/Kind.java

```diff
@@ -18,7 +18,8 @@
     FIXED_ASSETS_INVESTMENTS_NOTE("small-full-accounts-note#fixed-assets-investments"),
     EMPLOYEES_NOTE("small-full-accounts-note#employees"),
     CIC_REPORT("cic-report#cic-report"),
-    CIC34_REPORT("cic34-report#cic34-report");
+    CIC34_REPORT("cic34-report#cic34-report"),
```

Comment:
can we change this to take out the `34` ? leaving `cic-report#cic-report`

in the reworked spec we've avoided using the form number and will treat it as data going forwards so no need t include the form number
http://ch-jenkins-rh1.internal.ch:3123/manipulate-company-data/resources/cicreport?v=latest

Reply:
This enum was existing from work done before the spec changes and will be removed by my 'statements' changes 👍